src/ziggurat/metrics.clj

```diff
@@ -56,11 +56,12 @@
 
 (defn- inc-or-dec-count
   ([sign metric-namespace metric]
-   (inc-or-dec-count sign metric-namespace metric nil))
-  ([sign metric-namespaces metric additional-tags]
+   (inc-or-dec-count sign metric-namespace metric 1))
+  ([sign metric-namespaces metric n-or-additional-tags]
```

Comment:
what about the use case where we need to pass both additional tags and n??

Reply:
do we currently have that use case in production?

Reply:
so, we have the use case where people send n, we are now giving a functionality where people can also send additional tags. What if the said people want to send additional tags?
I don't understand why it needs to be an either-or case.  Why can't we support both?

Reply:
we have people who are sending n to increment/decrement count. Now, we have given the functionality to send additional tags, what will have to the said users who want to use this functionality? I don't understand why we need an either-or case for n or additional tags.
Why can't we support both?